src/Aspire.Hosting.Azure.CosmosDB/AzureCosmosDBExtensions.cs

```diff
@@ -136,11 +80,18 @@ public static IResourceBuilder<AzureCosmosDBResource> RunAsEmulator(this IResour
             cosmosClient = CreateCosmosClient(connectionString);
         });
 
+        // Use custom health check that also seeds the databases and containers
         var healthCheckKey = $"{builder.Resource.Name}_check";
-        builder.ApplicationBuilder.Services.AddHealthChecks().AddAzureCosmosDB(sp =>
```

Comment:
Do we still need a reference on `AspNetCore.HealthChecks.CosmosDb` if we aren't using it?

Reply:
Removed, replaced with `Microsoft.Azure.Cosmos` which was transitive before. Also added `NewtonSoft.Json` since there is a target that checks we include it. It was a dependency of `Microsoft.Azure.Cosmos` in 3.45.2.